src/v2/ion/IonResponseHelper.js

```diff
@@ -123,9 +123,13 @@ const getGlobalErrors = (res) => {
 const convertFormErrors = (response) => {
   let errors = {
     errorCauses: getRemediationErrors(response),
-    errorSummary: getGlobalErrors(response)
+    errorSummary: getGlobalErrors(response),
   };
 
+  if (response.messages?.value[0]?.i18n.key) {
```

Comment:
this has already been added in 5.3 https://github.com/okta/okta-signin-widget/commit/3c1559f00e8bfbde3e3da9f6f657b57e720832a0#diff-1a8fa5d91b81ac1f2b5a724f95f37a6cf2e065b8d78f82be8dffa5dd8f527260R140

Reply:
yeah i was looking for that change, but didn't knew it was merged to 5.3